docs/modules/ReaderTaskEither.ts.md

```diff
@@ -1062,6 +1063,25 @@ export declare function bracket<R, E, A, B>(
 
 Added in v2.0.4
 
+## filterOrElseW
+
+Less strict version of [`filterOrElse`](#filterOrElse).
+
+**Signature**
+
+```ts
+export declare const filterOrElseW: {
+  <A, B extends A, E2>(refinement: Refinement<A, B>, onFalse: (a: A) => E2): <R, E1>(
+    ma: ReaderTaskEither<R, E1, A>
```

Comment:
Can we widen the `R` too?

like the `chainW`?

Reply:
@mohaalak what do you mean? as opposed to `chainW` there's only one `R` involved here

Reply:
Oh yep I'm sorry I was thinking about orElse